tests/testthat/test_baseline.R

```diff
@@ -17,21 +17,25 @@ test_that("binomial evaluations are correct in baseline()", {
 
```

Comment:
Please revert the autoformatting. I get the urge to do formatting, but it makes reviewing changes difficult, not just here but also when regenerating test with xpectr and checking with a diff tool :)

Reply:
@IndrajeetPatil Did you see this comment? :)